README.md

```diff
@@ -10,6 +10,8 @@
 [![GoDoc](https://godoc.org/github.com/teamhanko/hanko?status.svg)](https://godoc.org/github.com/teamhanko/hanko)
 [![npm (scoped)](https://img.shields.io/npm/v/@teamhanko/hanko-elements?label=hanko-elements)](https://www.npmjs.com/package/@teamhanko/hanko-elements)
 [![npm (scoped)](https://img.shields.io/npm/v/@teamhanko/hanko-frontend-sdk?label=hanko-frontend-sdk)](https://www.npmjs.com/package/@teamhanko/hanko-frontend-sdk)
+<img src="https://img.shields.io/static/v1?label=Join&message=Discord&color=blueviolet&style=flat-square" alt="Join Discord">
+<img src="https://img.shields.io/twitter/follow/hanko_io?style=social" alt="Follow us on Twitter">
```

Comment:
Links are missing

Reply:
all issues resolved mentioned by you ,thank you